### DISCUSSION TITLE
GitHub Desktop: Unable to see whole screen/window cuts off necessary functions

### DISCUSSION THREAD

**Discussion by ghost** [community/community#38716] [Accessibility] [answered]
### Select Topic Area

Question

### Body

Hi, I require my computer to be zoomed and text enlarged to a certain level to due visual and neurological conditions. I'm trying to use GitHub desktop on Windows 10 on which everything is enlarged to 150% for me to be able to read content.

On my 24" monitor, the GitHub Desktop window is cut off at the bottom and there is no way for me to access the cut off content. For example, I would like to submit a Pull Request, but I suspect the button is cut off somewhere off the screen below the text box for "Description" in the screen capture below. 

I tried zooming out in GitHub Desktop but it doesn't zoom out any further than it already is. I tried toggling full screen view and that doesn't change anything either. I can't change my default Windows zoom level below 150% or I will not be able to read/use other applications.

![image](https://user-images.githubusercontent.com/94939137/200934483-11de1b65-8c14-41f8-9ef4-4823ffdbc7b9.png)

Can anyone offer some assistance to address this issue? It may be a significant usability feature lacking for individuals who are visually impaired. I am unable to use the software as intended with my limitations unless I am missing something obvious. Thanks!

**Comment by ghost** [comment, marked as answer]
FYI it seems updating the software resolved the issue.

**Comment by cehfisher** [comment]
Hi @edlabo - this issue appears to be resolved [per this comment](https://github.com/orgs/community/discussions/38716#discussioncomment-4101118). Could you please close this issue? Thank you! (cc @queenofcorgis)
